modules/apps/commerce/commerce-theme-minium/commerce-theme-minium/src/templates/portlet.ftl

```diff
@@ -66,12 +66,14 @@
 
 	<div class="${portlet_content_css_class}">
 		<#if portlet_display.isShowBackIcon()>
-			<a class="icon-monospaced portlet-icon-back text-default" href="${portlet_back_url}" title="<@liferay.language key="return-to-full-page" />">
-				<@liferay_ui["icon"]
+			<div class="icon-monospaced portlet-icon-back text-default" title="<@liferay.language key="return-to-full-page" />">
+				<@clay["button"]
+					displayType="unstyled"
 					icon="angle-left"
 					markupView="lexicon"
+					onClick="Liferay.Util.navigate('${portlet_back_url}')"
```

Comment:
I don't think this is the right fix, if the issue is that `portlet_back_url` could be a script, the correct fix would be to encode the value and make sure that a script wouldn't execute.

Plus, this doesn't necessarily fix the [issue at hand](https://liferay.atlassian.net/browse/LPS-195448), because any sort of "action" request URL could be provided. In order to fix this, we need to not allow for the `portlet_back_url` to be provided by the URL

Reply:
Okay @bryceosterhaus, thanks for your review I'll work on it